js/index.js

```diff
@@ -1,7 +1,47 @@
-window.onload = () => {
-  document.getElementById('start-button').onclick = () => {
-    startGame();
-  };
+/** @type {HTMLCanvasElement} */
 
-  function startGame() {}
-};
+const canvas = document.getElementById("canvas")
+
+const ctx = canvas.getContext('2d');
+
+const startButton = document.getElementById('start-button');
+
+const player = new Component(carImg, 0, 200, 75, 75, "blue", ctx);
```

Comment:
If you check the `component.js` file you can see that the class `Component` accepts the x, y, w, h, and the ctx:
```
class Component {
  constructor(x, y, w, h, ctx) {
(...)
```

But here when you're creating an instance of the class `Component` you are passing:
```
const player = new Component(carImg, 0, 200, 75, 75, "blue", ctx);
```

So what's happening is that:
 • `carImg` will be interpreted as the `x`, 
 • the `0` will be the `y`, 
 • the `200` will be the `w`, 
 • the `75` will be the `h`, 
 • the other `75` will be the `ctx`, 
 • `'blue'` and `ctx` will not be read because the `component` is expecting only 5 `arguments` and you're passing 7. 

The correct way to create the instance of the class `Component` that will be the car is:
```
const player = new Component(0, 200, 75, 75, ctx);
```
If you make this change your car will appear!